### PR TITLE
Adding FERPlus model to model zoo

### PR DESCRIPTION
**Pull request by lupesko**
*Description of changes:*
Adding FERPlus model (emotion detection CNN) to model zoo, this model is imported from ONNX

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

@aaronmarkham @ankkhedia

### CODE REVIEW

**Comment by ankkhedia** [comment]
@lupesko The integration test was written when we had only CNN models trained on imagenet and it is not generic enough for all types of inputs/outputs. 
There is an open issue for a generic framework which needs to be taken up. 
Issue #340

File: mms/tests/integration_tests/test_utils.py [hunk 1]
```diff
@@ -44,6 +44,7 @@
     'resnext-101-64x4d': 'https://s3.amazonaws.com/model-server/models/resnext-101-64x4d/resnext-101-64x4d.model',
     'squeezenet_v1.1': 'https://s3.amazonaws.com/model-server/models/squeezenet_v1.1/squeezenet_v1.1.model',
     'vgg19': 'https://s3.amazonaws.com/model-server/models/vgg19/vgg19.model',
+    'ferplus': 'https://s3.amazonaws.com/model-server/models/FERPlus/FERPlus.model',
```
Comment (aaronmarkham) [inline mms/tests/integration_tests/test_utils.py]: This won't work. If the data input is `Input2505` for the input layer, the test utils only expects to see `data` or `input0`. I noticed this with K's keras blog - it used `/input11` or something. I thought the slash was a problem and the input layer deviation was super interesting. Really need to think through the implications of this for the zoo, mms, and how stuff is tested.

Comment (lupesko) [reply mms/tests/integration_tests/test_utils.py]: Yes, note my comments above. The integration suite code needs to be refactored to be able and handle broader set of models.

**Comment by lupesko** [comment]
@aaronmarkham @ankkhedia updated the PR to include only the added model.
Will add test improvements later as part of #340 
Please review.

**Comment by aaronmarkham** [comment]
Lgtm

**Comment by ankkhedia** [comment]
PR looks good!

**Pull request merged**